tools/arxiv/orchestration/orchestrator.py

```
@@ -0,0 +1,482 @@
+    start_year: int = 2010
+    end_year: int = 2025
+    categories: List[str] = field(default_factory=lambda: ["cs.LG", "cs.CL", "cs.CV"])
+    keywords: Optional[str] = None
+    target_count: Optional[int] = None
+    cap_mode: str = "per-year"
+    require_pdf: bool = False
+    
+    @classmethod
+    def from_yaml(cls, path: str) -> SearchConfiguration:
+        """Load configuration from YAML file."""
+        with open(path, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)